back/app/models/concerns/user_roles.rb

```diff
@@ -131,34 +131,6 @@ def admin?
     roles.any? { |r| r['type'] == 'admin' }
   end
 
-  # Reduce the roles to add to the JWT cookie to stop it getting too large
-  # Roles from cookie are ONLY used by workshops and cl2-admin-templates
```

Comment:
Although this comment says workshops uses roles from cookies, I am pretty sure it does not. instead it uses the user ID from the JWT payload, then makes an api request to get the user attributes, and only then check the roles it finds in those attributes.

Reply:
Pretty sure or certain?